Update ANGLE to 2024-02-07 (f431641a948660f5e1709aa7cd89b16ccf93f1de)

kkinnunen-apple opened the pull request:
#### c97ff4f316728fe1444b50ad225f0389e0b69cd1
<pre>
Update ANGLE to 2024-02-07 (f431641a948660f5e1709aa7cd89b16ccf93f1de)
<a href="https://bugs.webkit.org/show_bug.cgi?id=268926">https://bugs.webkit.org/show_bug.cgi?id=268926</a>
<a href="https://rdar.apple.com/problem/122481338">rdar://problem/122481338</a>

Reviewed by David Kilzer and Dan Glastonbury

Contains upstream commits:
git log --oneline eb77635a5d3f45228513faf2752c1ab422114312..f431641a948660f5e1709aa7cd89b16ccf93f1de --pretty=%h %s
f431641a94 Revert &quot;Vulkan: Get rid of X11 include in DisplayVkXcb.cpp&quot;
ab71d75104 Fix an assert on multidim array constructors
8abbe2836b Vulkan: Get rid of X11 include in DisplayVkXcb.cpp
98d3f32722 Change python import conditions to match dawn&apos;s.
cf77126a7f Turn off ADC explicitly
e62bd70a6c Metal: Disable Metal on older Mac models with GPU family 1.
d05c9a5ee6 Frontend support for QCOM foveated extensions
11eb5eb7ac Traces: support checkout of individual traces
9d165231e1 Roll Chromium from 37f9e1b14cf7 to c1ca24b91ed5 (609 revisions)
5a0615588a Vulkan: Update dynamic buffer size policy
efd35e3dfe Roll VK-GL-CTS from 1aeae2d1d57e to b3344240e7fc (2 revisions)
39b9d94750 Suppress VUID-VkGraphicsPipelineCreateInfo-dynamicRendering-06576
f5f3304a2b Vulkan: Simplify handling of YUV filtering support
d5720c8653 Reland &quot;Add third_party/perfetto&quot;
c70c329e10 Vulkan: Fix texture copy vs pre-rotation
ffe758b16b Roll VK-GL-CTS from fe0d0f90469f to 1aeae2d1d57e (11 revisions)
82ba79ffea OpenCL: Update CLtypes.h to cl_types.h
0c3a6aa6dc Roll SwiftShader from d040a5bab638 to eb75201a4e03 (5 revisions)
a45a6c1d31 Roll Chromium from 861058c0b131 to 37f9e1b14cf7 (513 revisions)
f8c06f103a OpenCL: Add types to cl_mem origin and region
ecc35205ee Move uniform block dirty bits to State
ea1cea778c Roll Chromium from d3635c12e43a to 861058c0b131 (574 revisions)
1abfdc3732 Remove suppression of VVL error
6340ed68d9 Revert &quot;Add third_party/perfetto&quot;
157088ad5f Add EGL extension for WebGPU
a70b6f56fd Add support for WinUI3/Windows App SDK
0640f65195 USE GCE Credentials for Chrome Bot
be8d1c2467 Roll Chromium from 84709e977bd1 to d3635c12e43a (585 revisions)
0ada0b7f1f Test: Add Plague Inc. trace
b514d7733a Manual roll vulkan-deps from 97f44eb2b9df to 3834da2004ec (22 revisions)
dcffb53ec1 Revert &quot;Clean up ANGLE on system build configuration.&quot;
9ca81c7e65 FrameCapture: Handle zero vertex count draws
98846aa8f9 Metal: Temporarily disable asm injection.
e849802953 Roll Chromium from 0fdef08dd9c7 to 84709e977bd1 (621 revisions)
ab4aed3fce Bugfix in PixelLocalStorageTest
5c2fc02b8d Vulkan: Bugfix in ImageFormat
349e75cc37 Suppress VUID-VkFramebufferAttachmentImageInfo-viewFormatCount-09536
0c4d644649 Rework uniform block &lt;-&gt; uniform buffer mapping
4b76bead8f Add third_party/perfetto
4a5abf6d06 Add Dawn checkout condition to .gitmodules
98e0266851 Clean up ANGLE on system build configuration.
c6d827e445 Disable the Context shared lock on Android for Chrome.
3fdf576a0c Vulkan: Workaround Nvidia driver bug with OpSelect
0ea24ef2eb Unsuppress passing tests
863d48627e Work around MSAN complaint with pool allocator - Part 2
3fe4252740 OpenCL: Fix input -&gt; inputs for clspv_c_strings
351b85f222 Roll SwiftShader from 3bc9ccd923da to d040a5bab638 (1 revision)
9545bdbd7f Roll Chromium from 70e6bcc15839 to 0fdef08dd9c7 (58 revisions)
2f4f3dcfb8 Manual roll Chromium from 4d771baa2602 to 70e6bcc15839 (609 revisions)
9318a143b0 Add __asm__(&quot;&quot;) statements to loop bodies in MSL.
0beed7535d Work around MSAN complaint with pool allocator
6367f5415e Vulkan: supply YcbcrConversionDesc earlier
eaf15b354f Manual roll VK-GL-CTS from 03517cba39ae to fe0d0f90469f (2 revisions)
66517b0b02 Revert &quot;Use ADC for GCE VM&quot;
f405b99905 Make RefCountObject use atomics for the reference count
6c23a4230b Tests: skip one of dEQP texture_border_clamp tests on SwS
02aa605af8 Metal: Include only the &lt;metal_stdlib&gt; header
8034d09c11 Manual roll VK-GL-CTS from 0f6fd3de8110 to 03517cba39ae (14 revisions)
36013d998a Improve build for UWP/Windows 10
cbdbe3bd6e deqp: Unused test expectations do not cause failure, only log
5fb6cfa051 Roll vulkan-deps from 52f65da37e07 to 97f44eb2b9df (17 revisions)
f00263e282 Roll Chromium from 60c7e3d68ae8 to 4d771baa2602 (575 revisions)
e9bd169c05 Vulkan: Fix nullptr in Diagnostics
d0f5a66208 Bugfix in entry points for QCOM foveated extensions
d20c693a84 Android.bp: add empty lines as bpfmt wants
2dfbedee7d Use ADC for GCE VM
127dabe556 Android.bp: Move cflags repeated in each target to cc_defaults
6fcca069eb Add a note on why ANGLE is pinned to C++17
c7ed7da640 Roll vulkan-deps from a2b38a7bbea9 to 52f65da37e07 (8 revisions)
7d30cf9f69 Metal: Simplify sign function translation
52527b8fd1 Roll Chromium from 5bfb2faa963e to 60c7e3d68ae8 (652 revisions)
8e2678463d gn: Enable clspv build on angle_enable_cl
4761bd7c14 Manual roll vulkan-deps from 167fd9d49add to a2b38a7bbea9 (17 revisions)
ac71a59239 Vulkan: updates to pipeline cache graph dumping logic
2978450af1 Vulkan: add etc to bc transcoding perf test.
b007c74dcd GL: Separate dirty bits leading to glUniformBlockBinding
66de849281 GL: Fix UBO binding mapping change
886d0260cf Don&apos;t pull Dawn in chromium builds
5857e0d63b Metal: Adjust raster order groups AMD workaround condition
a76e279851 Suppress VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912
2c8784c9ff Roll Chromium from 2e1050880b38 to 5bfb2faa963e (609 revisions)
600a1a80da Metal: Clamp vector element index references
1100cc5c72 Rename ShaderVariableBuffer
81a43bd703 Tests: skip TexelFetchLodOutOfBounds on SwS
3f56817745 Add third_party/dawn to .gitignore
4cda949b4b Trace tests: extend warmup to at least 1.5s
755372c87a Tests: skip UniformUsageCombinations test on SwS
f794c9c2bb Tests for out-of-bounds LOD in texelFetch
43ecc0312a Roll SwiftShader from 5ab5177fc72d to 3bc9ccd923da (1 revision)
9519348b86 Roll Chromium from 6d5cc7186844 to 2e1050880b38 (298 revisions)
341906d568 Vulkan: Never delay device and queue selection
2431d4dfc9 Metal: Skip array index clamp for constant values
6975fbdfba Manual roll Chromium from 32cda1ad29f0 to 6d5cc7186844 (1322 revisions)
2ba5bb12ab android_helper: fix corner-case byte/string mismatch
c6fbf93dec Vulkan: Fix input attachments leaking into uniform list
317108d6ac Vulkan: Enable recordable bit for RGBA8 and RGB10_A2
f43db42ccf Enable LogGles32Capabilities test for vulkan backend
16a5e8f133 Revert &quot;Prefer linear filtering for YUV for Pixel6+&quot;
3217caa1b9 Roll SwiftShader from 328e794f0c8b to 5ab5177fc72d (1 revision)
e27af543b6 Roll vulkan-deps from c00c99b8e979 to 167fd9d49add (6 revisions)
151c157198 Vulkan: Clean up RGB8-to-RGBA8 ubyte function
fc742305e0 Metal: Simplify geometry functions translation
8a47d6b14f Roll VK-GL-CTS from 04d7f781e691 to 0f6fd3de8110 (15 revisions)
175514c64e Translator: Bundle metadata flags coming out of the translator
07e41ef0e0 Roll vulkan-deps from 9b8079729ca3 to c00c99b8e979 (2 revisions)
0da6522524 Prefer linear filtering for YUV for Pixel6+
291490e291 Manual roll Chromium from 84fab30ecd66 to 32cda1ad29f0 (2747 revisions)
dd936c2314 Android: Build angle libs with a shared copy of libz
f2c00e8c98 Adding Dawn stub backend to ANGLE
fad2adb209 Vulkan: Fix importing external object and using as storage image
f1150231f1 Track WebGL buffer type in gl::Buffer state.
40b6739712 Roll vulkan-deps from 5803e963cb95 to 9b8079729ca3 (7 revisions)
87fb3bf312 Roll SwiftShader from f4819d2276b7 to 328e794f0c8b (1 revision)
d093c3d3c2 Add entry points for QCOM foveated extensions
42d75c7e99 Metal: Fix raster order group AMD workaround
ed2a8ef865 Vulkan: Defer QFOT when acquiring texture with GL_NONE layout
05cd0b4c20 Tests: Add Asphalt 9 2024 trace
b00481fc58 Roll vulkan-deps from 41ee13a742db to 5803e963cb95 (7 revisions)
2e167e0922 Adding Dawn
3a33cbb029 Test for making context current before surface creation
91c981c503 Robust shader compile after shader binary is rejected
eedf3e9e9e Metal: Limit raster order groups workaround to affected GPUs
efeb6570af Suppress ParallelLinkProgramBenchmark.Run/gl_* on Linux
7bc0fa21d2 Roll vulkan-deps from b8c02f610c1d to 41ee13a742db (14 revisions)
572323ccbb Fix program link after backend rejects program binary
e9aa41f567 Trace/Replay: Add multi-context call grouping
3461be1ae8 Version check usages of getPackageInfo() and of()
7d9c1d5fb0 Add a test to check eglInitialize works in multithread
aa309a65e3 Roll VK-GL-CTS from 75a6a8315e20 to 04d7f781e691 (13 revisions)
250b2f681d Add imports for use_resource_processor: true
a06f5f892e Android: Apply rules to global settings on boot
99ca7f240b Android: Add ANGLE to rules so it is non-empty
28290eae4b Metal: Do not try to create buffers exceeding device limits
126d7b10bf Roll vulkan-deps from 124ff11a5e15 to b8c02f610c1d (6 revisions)

Canonical link: <a href="https://commits.webkit.org/274279@main">https://commits.webkit.org/274279@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/717cbc28005895c3f36a20fed347f01993027297

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/38378 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/17321 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40714 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/40935 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/34122 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/20220 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/14665 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/32344 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/38951 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14694 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/33446 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12718 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/12748 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/34295 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/42214 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/34976 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/34709 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38535 "Passed tests") | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/13286 "Build is being retried. Recent messages:") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/10996 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36734 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/14851 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8650 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13762 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/14317 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ddkilzer reviewed (approved):
rs=me, but maybe you want someone else to review this?

djg reviewed (approved):
:shipit:

kkinnunen-apple committed:
Update ANGLE to 2024-02-07 (f431641a948660f5e1709aa7cd89b16ccf93f1de)
https://bugs.webkit.org/show_bug.cgi?id=268926
rdar://problem/122481338

Reviewed by David Kilzer and Dan Glastonbury

Contains upstream commits:
git log --oneline eb77635a5d3f45228513faf2752c1ab422114312..f431641a948660f5e1709aa7cd89b16ccf93f1de --pretty=%h %s
f431641a94 Revert "Vulkan: Get rid of X11 include in DisplayVkXcb.cpp"
ab71d75104 Fix an assert on multidim array constructors
8abbe2836b Vulkan: Get rid of X11 include in DisplayVkXcb.cpp
98d3f32722 Change python import conditions to match dawn's.
cf77126a7f Turn off ADC explicitly
e62bd70a6c Metal: Disable Metal on older Mac models with GPU family 1.
d05c9a5ee6 Frontend support for QCOM foveated extensions
11eb5eb7ac Traces: support checkout of individual traces
9d165231e1 Roll Chromium from 37f9e1b14cf7 to c1ca24b91ed5 (609 revisions)
5a0615588a Vulkan: Update dynamic buffer size policy
efd35e3dfe Roll VK-GL-CTS from 1aeae2d1d57e to b3344240e7fc (2 revisions)
39b9d94750 Suppress VUID-VkGraphicsPipelineCreateInfo-dynamicRendering-06576
f5f3304a2b Vulkan: Simplify handling of YUV filtering support
d5720c8653 Reland "Add third_party/perfetto"
c70c329e10 Vulkan: Fix texture copy vs pre-rotation
ffe758b16b Roll VK-GL-CTS from fe0d0f90469f to 1aeae2d1d57e (11 revisions)
82ba79ffea OpenCL: Update CLtypes.h to cl_types.h
0c3a6aa6dc Roll SwiftShader from d040a5bab638 to eb75201a4e03 (5 revisions)
a45a6c1d31 Roll Chromium from 861058c0b131 to 37f9e1b14cf7 (513 revisions)
f8c06f103a OpenCL: Add types to cl_mem origin and region
ecc35205ee Move uniform block dirty bits to State
ea1cea778c Roll Chromium from d3635c12e43a to 861058c0b131 (574 revisions)
1abfdc3732 Remove suppression of VVL error
6340ed68d9 Revert "Add third_party/perfetto"
157088ad5f Add EGL extension for WebGPU
a70b6f56fd Add support for WinUI3/Windows App SDK
0640f65195 USE GCE Credentials for Chrome Bot
be8d1c2467 Roll Chromium from 84709e977bd1 to d3635c12e43a (585 revisions)
0ada0b7f1f Test: Add Plague Inc. trace
b514d7733a Manual roll vulkan-deps from 97f44eb2b9df to 3834da2004ec (22 revisions)
dcffb53ec1 Revert "Clean up ANGLE on system build configuration."
9ca81c7e65 FrameCapture: Handle zero vertex count draws
98846aa8f9 Metal: Temporarily disable asm injection.
e849802953 Roll Chromium from 0fdef08dd9c7 to 84709e977bd1 (621 revisions)
ab4aed3fce Bugfix in PixelLocalStorageTest
5c2fc02b8d Vulkan: Bugfix in ImageFormat
349e75cc37 Suppress VUID-VkFramebufferAttachmentImageInfo-viewFormatCount-09536
0c4d644649 Rework uniform block <-> uniform buffer mapping
4b76bead8f Add third_party/perfetto
4a5abf6d06 Add Dawn checkout condition to .gitmodules
98e0266851 Clean up ANGLE on system build configuration.
c6d827e445 Disable the Context shared lock on Android for Chrome.
3fdf576a0c Vulkan: Workaround Nvidia driver bug with OpSelect
0ea24ef2eb Unsuppress passing tests
863d48627e Work around MSAN complaint with pool allocator - Part 2
3fe4252740 OpenCL: Fix input -> inputs for clspv_c_strings
351b85f222 Roll SwiftShader from 3bc9ccd923da to d040a5bab638 (1 revision)
9545bdbd7f Roll Chromium from 70e6bcc15839 to 0fdef08dd9c7 (58 revisions)
2f4f3dcfb8 Manual roll Chromium from 4d771baa2602 to 70e6bcc15839 (609 revisions)
9318a143b0 Add __asm__("") statements to loop bodies in MSL.
0beed7535d Work around MSAN complaint with pool allocator
6367f5415e Vulkan: supply YcbcrConversionDesc earlier
eaf15b354f Manual roll VK-GL-CTS from 03517cba39ae to fe0d0f90469f (2 revisions)
66517b0b02 Revert "Use ADC for GCE VM"
f405b99905 Make RefCountObject use atomics for the reference count
6c23a4230b Tests: skip one of dEQP texture_border_clamp tests on SwS
02aa605af8 Metal: Include only the <metal_stdlib> header
8034d09c11 Manual roll VK-GL-CTS from 0f6fd3de8110 to 03517cba39ae (14 revisions)
36013d998a Improve build for UWP/Windows 10
cbdbe3bd6e deqp: Unused test expectations do not cause failure, only log
5fb6cfa051 Roll vulkan-deps from 52f65da37e07 to 97f44eb2b9df (17 revisions)
f00263e282 Roll Chromium from 60c7e3d68ae8 to 4d771baa2602 (575 revisions)
e9bd169c05 Vulkan: Fix nullptr in Diagnostics
d0f5a66208 Bugfix in entry points for QCOM foveated extensions
d20c693a84 Android.bp: add empty lines as bpfmt wants
2dfbedee7d Use ADC for GCE VM
127dabe556 Android.bp: Move cflags repeated in each target to cc_defaults
6fcca069eb Add a note on why ANGLE is pinned to C++17
c7ed7da640 Roll vulkan-deps from a2b38a7bbea9 to 52f65da37e07 (8 revisions)
7d30cf9f69 Metal: Simplify sign function translation
52527b8fd1 Roll Chromium from 5bfb2faa963e to 60c7e3d68ae8 (652 revisions)
8e2678463d gn: Enable clspv build on angle_enable_cl
4761bd7c14 Manual roll vulkan-deps from 167fd9d49add to a2b38a7bbea9 (17 revisions)
ac71a59239 Vulkan: updates to pipeline cache graph dumping logic
2978450af1 Vulkan: add etc to bc transcoding perf test.
b007c74dcd GL: Separate dirty bits leading to glUniformBlockBinding
66de849281 GL: Fix UBO binding mapping change
886d0260cf Don't pull Dawn in chromium builds
5857e0d63b Metal: Adjust raster order groups AMD workaround condition
a76e279851 Suppress VUID-vkCmdEndDebugUtilsLabelEXT-commandBuffer-01912
2c8784c9ff Roll Chromium from 2e1050880b38 to 5bfb2faa963e (609 revisions)
600a1a80da Metal: Clamp vector element index references
1100cc5c72 Rename ShaderVariableBuffer
81a43bd703 Tests: skip TexelFetchLodOutOfBounds on SwS
3f56817745 Add third_party/dawn to .gitignore
4cda949b4b Trace tests: extend warmup to at least 1.5s
755372c87a Tests: skip UniformUsageCombinations test on SwS
f794c9c2bb Tests for out-of-bounds LOD in texelFetch
43ecc0312a Roll SwiftShader from 5ab5177fc72d to 3bc9ccd923da (1 revision)
9519348b86 Roll Chromium from 6d5cc7186844 to 2e1050880b38 (298 revisions)
341906d568 Vulkan: Never delay device and queue selection
2431d4dfc9 Metal: Skip array index clamp for constant values
6975fbdfba Manual roll Chromium from 32cda1ad29f0 to 6d5cc7186844 (1322 revisions)
2ba5bb12ab android_helper: fix corner-case byte/string mismatch
c6fbf93dec Vulkan: Fix input attachments leaking into uniform list
317108d6ac Vulkan: Enable recordable bit for RGBA8 and RGB10_A2
f43db42ccf Enable LogGles32Capabilities test for vulkan backend
16a5e8f133 Revert "Prefer linear filtering for YUV for Pixel6+"
3217caa1b9 Roll SwiftShader from 328e794f0c8b to 5ab5177fc72d (1 revision)
e27af543b6 Roll vulkan-deps from c00c99b8e979 to 167fd9d49add (6 revisions)
151c157198 Vulkan: Clean up RGB8-to-RGBA8 ubyte function
fc742305e0 Metal: Simplify geometry functions translation
8a47d6b14f Roll VK-GL-CTS from 04d7f781e691 to 0f6fd3de8110 (15 revisions)
175514c64e Translator: Bundle metadata flags coming out of the translator
07e41ef0e0 Roll vulkan-deps from 9b8079729ca3 to c00c99b8e979 (2 revisions)
0da6522524 Prefer linear filtering for YUV for Pixel6+
291490e291 Manual roll Chromium from 84fab30ecd66 to 32cda1ad29f0 (2747 revisions)
dd936c2314 Android: Build angle libs with a shared copy of libz
f2c00e8c98 Adding Dawn stub backend to ANGLE
fad2adb209 Vulkan: Fix importing external object and using as storage image
f1150231f1 Track WebGL buffer type in gl::Buffer state.
40b6739712 Roll vulkan-deps from 5803e963cb95 to 9b8079729ca3 (7 revisions)
87fb3bf312 Roll SwiftShader from f4819d2276b7 to 328e794f0c8b (1 revision)
d093c3d3c2 Add entry points for QCOM foveated extensions
42d75c7e99 Metal: Fix raster order group AMD workaround
ed2a8ef865 Vulkan: Defer QFOT when acquiring texture with GL_NONE layout
05cd0b4c20 Tests: Add Asphalt 9 2024 trace
b00481fc58 Roll vulkan-deps from 41ee13a742db to 5803e963cb95 (7 revisions)
2e167e0922 Adding Dawn
3a33cbb029 Test for making context current before surface creation
91c981c503 Robust shader compile after shader binary is rejected
eedf3e9e9e Metal: Limit raster order groups workaround to affected GPUs
efeb6570af Suppress ParallelLinkProgramBenchmark.Run/gl_* on Linux
7bc0fa21d2 Roll vulkan-deps from b8c02f610c1d to 41ee13a742db (14 revisions)
572323ccbb Fix program link after backend rejects program binary
e9aa41f567 Trace/Replay: Add multi-context call grouping
3461be1ae8 Version check usages of getPackageInfo() and of()
7d9c1d5fb0 Add a test to check eglInitialize works in multithread
aa309a65e3 Roll VK-GL-CTS from 75a6a8315e20 to 04d7f781e691 (13 revisions)
250b2f681d Add imports for use_resource_processor: true
a06f5f892e Android: Apply rules to global settings on boot
99ca7f240b Android: Add ANGLE to rules so it is non-empty
28290eae4b Metal: Do not try to create buffers exceeding device limits
126d7b10bf Roll vulkan-deps from 124ff11a5e15 to b8c02f610c1d (6 revisions)

Canonical link: https://commits.webkit.org/274279@main
